docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '3.1'
 services:
   mysql:
     image: mysql:5.6
+    platform: linux/x86_64
```

Comment:
In #23, this value is something different. I don't have an M1 machine, so I'm not sure what this should be, but wanted to call it out.

Reply:
@anthony-liddle I'm updating to the other value, can you double-check on your machine if it still works? thanks aly